package.json

```diff
@@ -96,6 +96,11 @@
           "type": "string",
           "default": "",
           "description": "A root message for dotnet test command. In English it is 'The following Tests are available:' but for example in Polish it is 'Dostępne są następujące testy:'"
+        },
+        "dotnet-test-explorer.useTreeView": {
+          "type": "boolean",
+          "default": false,
```

Comment:
I figured it would be best to have users opt into the new behavior, but if you think it makes more sense to have it on by default I see no issue with that